test/trimCharsStart.js

```diff
@@ -0,0 +1,33 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('trimCharsStart', function() {
+  it('should remove all occurences of a trim character', function() {
+    assert.strictEqual(RA.trimCharsStart('-')('---abc'), 'abc');
+  });
+
+  it('should remove all occurences of all trim characters', function() {
+    assert.strictEqual(RA.trimCharsStart('_-')('-_-abc'), 'abc');
+  });
+
+  it('should not remove the trim characters from the end of the string', function() {
+    assert.strictEqual(RA.trimCharsStart('_-')('-_-abc-_-'), 'abc-_-');
+  });
+
+  it('should return the target string as-is if none of the trim characters are present', function() {
```

Comment:
If test descriptions gets long it's very hard to read them. We have a convention to fix that
```js
context('given none of the trim characters are present', function() {
  specify('should return the target string as-is ', function() {
    assert.strictEqual(RA.trimCharsStart('_-', 'abc'), 'abc');
  });        
});

Reply:
Usualy the long sentences cotains logical branching (if/when). Please process the rest of the tests as I described above.